features/input.yml

```diff
@@ -77,31 +78,14 @@ compat_features:
   # Excluded because of a non-real value
   # - html.elements.input.size
   - html.elements.input.src
-  - html.elements.input.step
   - html.elements.input.type_button
   - html.elements.input.type_checkbox
-  - html.elements.input.type_color
-  - html.elements.input.type_date
-  - html.elements.input.type_datetime-local
-  - html.elements.input.type_email
   - html.elements.input.type_file
   - html.elements.input.type_hidden
   - html.elements.input.type_image
-  # Excluded because of a non-real value
-  # - html.elements.input.type_month
-  - html.elements.input.type_number
   - html.elements.input.type_password
   - html.elements.input.type_password.insecure_login_handling
   - html.elements.input.type_radio
-  - html.elements.input.type_range
-  # Excluded because of a non-real value
-  # - html.elements.input.type_range.tick_marks
-  # - html.elements.input.type_range.vertical_orientation
   - html.elements.input.type_reset
   - html.elements.input.type_submit
```

Comment:
I think all the remaining `type_` features can be broken out:

- input-button
- input-checkbox
- input-file
- input-hidden
- input-image
- input-password
- input-radio
- input-reset-submit (I'd group these together, but feel free to separate if you disagree)

`type_text` should remain here.

Reply:
i kept reset and submit separate since they do different things, but also because reset is destructive to local data. not sure if we've given warnings about behavior like that in the work so far?